README.md

```diff
@@ -19,18 +19,18 @@ I'm using this state-machine as a simple replacement for Redux in some ReactJs b
 
 ## Get it
 
-  ```script
```

Comment:
There are a lot of formatting changes. Can you revert these to make the PR more directed towards your changes?

Reply:
Whoops! I sometimes forget I have `formatOnSave` turned on and not everyone wants the same format as me :)

I'll redo the pull when I get a chance.